core/src/dird/dird_conf.cc

```diff
@@ -186,31 +187,42 @@ static ResourceItem profile_items[] = {
   {nullptr, 0, 0, nullptr, 0, 0, nullptr, nullptr, nullptr}
 };
 
+#define USER_ACL(resource) \
+  { "JobACL", CFG_TYPE_ACL, ITEM(resource, user_acl.ACL_lists), Job_ACL, 0, NULL, NULL, NULL },\
+  { "ClientACL", CFG_TYPE_ACL, ITEM(resource, user_acl.ACL_lists), Client_ACL, 0, NULL, NULL, NULL },\
+  { "StorageACL", CFG_TYPE_ACL, ITEM(resource, user_acl.ACL_lists), Storage_ACL, 0, NULL, NULL, NULL },\
+  { "ScheduleACL", CFG_TYPE_ACL, ITEM(resource, user_acl.ACL_lists), Schedule_ACL, 0, NULL, NULL, NULL },\
+  { "RunACL", CFG_TYPE_ACL, ITEM(resource, user_acl.ACL_lists), Run_ACL, 0, NULL, NULL, NULL },\
+  { "PoolACL", CFG_TYPE_ACL, ITEM(resource, user_acl.ACL_lists), Pool_ACL, 0, NULL, NULL, NULL },\
+  { "CommandACL", CFG_TYPE_ACL, ITEM(resource, user_acl.ACL_lists), Command_ACL, 0, NULL, NULL, NULL },\
+  { "FileSetACL", CFG_TYPE_ACL, ITEM(resource, user_acl.ACL_lists), FileSet_ACL, 0, NULL, NULL, NULL },\
+  { "CatalogACL", CFG_TYPE_ACL, ITEM(resource, user_acl.ACL_lists), Catalog_ACL, 0, NULL, NULL, NULL },\
+  { "WhereACL", CFG_TYPE_ACL, ITEM(resource, user_acl.ACL_lists), Where_ACL, 0, NULL, NULL, NULL },\
+  { "PluginOptionsACL", CFG_TYPE_ACL, ITEM(resource, user_acl.ACL_lists), PluginOptions_ACL, 0, NULL, NULL, NULL },\
+  { "Profile", CFG_TYPE_ALIST_RES, ITEM(resource, user_acl.profiles), R_PROFILE, 0, NULL, "14.2.3-",\
+     "Profiles can be assigned to a Console. ACL are checked until either a deny ACL is found or an allow ACL. "\
+     "First the console ACL is checked then any profile the console is linked to." }
+
```

Comment:
How about using the USER_ACL (except "Profile") also for profile_items?

Reply:
Could you please verify, that "configure add user name=test ..." works?

Reply:
> Could you please verify, that "configure add user name=test ..." works?
Looks like. But it does not create a folder "User" if it does not already exist. Is this the itended behaviour?

Reply:
> How about using the USER_ACL (except "Profile") also for profile_items?

I would deny. They do not share the same items (RunACL, profiles) and in the resource objects they refer to different C-structures. 

The USER_ACL Macro was introduced to bring together console and user resource because they must be able to use the same interface to access the data.

Reply:
> > Could you please verify, that "configure add user name=test ..." works?
> 
> Looks like. But it does not create a folder "User" if it does not already exist. Is this the itended behaviour?

Yes, the user folder should be generated by the package.

Reply:
- enhanced using bconsole-pam test by configure add user name=user3
- rewrite macro USER_ACL to use in all relevant initializer lists (RunACL was missing in the one mentioned above)